content/docs/start/experiments.md

```diff
@@ -211,12 +211,257 @@ scores.json  auc       0.61314  0.07139
 And finally, we can compare `ROC` curves with a single command!
 
 ```dvc
+$ dvc plots modify prc.json -x recall -y precision
+$ git commit -am "Update plots format"
+
 $ dvc plots diff
 file:///Users/dvc/example-get-started/plots.html
 ```
 
 ![](/img/plots_roc_get_started.svg)
 
+<details>
+
+### 💡 Expand to see what plots modify does under the hood
```

Comment:
some duplication happening in the text below?

Reply:
Yes, something is wrong. It looks like over 200 lines were added and the `<details>` may not be closed properly.

Reply:
But do we even want to add all this info about `plots modify`? See also https://github.com/iterative/dvc.org/pull/1514#pullrequestreview-441099599 below.

Reply:
yeah, it should be short and simple with a link to the CR for the details

Reply:
p.s. @xnutsive you can sanity check in https://dvc-landing-xnutsive-ge-1visdu.herokuapp.com/doc/start/experiments. This is how it looks rn:

![image](https://user-images.githubusercontent.com/1477535/86279073-0cf81300-bb9f-11ea-9a77-233649be0c1a.png)